Week3/homework/step2-5.js

```diff
@@ -6,6 +6,12 @@ function multiplyAll(arr) {
 
   // Replace this comment and the next line with your code
   console.log(arr, product);
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      console.log(arr[i][j]);
```

Comment:
We don't need all these `console.log` statements in a pull request. It's fine when you are debugging your code, but once your code is ready you should remove them before sending in the pull request. Note that a pull request is a request to another developer to review your code. The console.log's are of no interest to the reviewer; they are just unnecessary noise (same applies to commented-out code, but so far, I haven't seen that in your pull request).